pkg/apis/monitoring/v1/thanos_types.go

```diff
@@ -287,6 +287,11 @@ type ThanosRulerSpec struct {
 	// +kubebuilder:default:="15s"
 	EvaluationInterval Duration `json:"evaluationInterval,omitempty"`
 
+	// Minimum amount of time to wait before resending an alert to Alertmanager.
+	// +kubebuilder:default:="1m"
```

Comment:
let's avoid overriding the default value from Thanos.
```suggestion
```

Reply:
Done, thanks